### PR TITLE
Update repo setup for watcher

### PR DESCRIPTION
**Pull request by ngreenwald**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Updates the setup docs, readme, and paths for basic watcher functionality

**How did you implement your changes**

Standardized the naming scheme for output paths across the notebooks. Also added in the folder creation logic to initial notebook. 

**Remaining issues**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +23,16 @@ The repo has four main parts, with associated code and jupyter notebooks for eac
...
+
+- 3a: real time monitoring. The [MIBI monitoring](./templates/3a_monitor_MIBI_run.ipynb) notebook will monitor an ongoing MIBI run, and begin processing the image data as soon as it is generated. This notebook is being continually be updated as we move more of our processing pipeline to happen in real time as the data is generated.
+- 3b: post-run monitoring. For each step in the monitoring notebook, we have a dedicated notebook that can perform the same tasks once a run is complete. This includes [the image extraction notebook](./templates/extract_bin_file.ipynb) and the [qc metrics notebook](./templates/3b_generate_qc_metrics.ipynb). 
+
 
 ### 4. Processing MIBI data
 Once your run has finished, you can begin to process the data to make it ready for analysis. To remove background signal contamination, as well as compensate for channel crosstalk, you can use the [compensation](./templates/4_compensate_image_data.ipynb) notebook. This will guide you through the process of determining the correct coefficients for the Rosetta algorith, which uses a flow-cytometry style compensation approach to remove spurious signal.
```
Comment (srivarra) [inline README.md:38]: Small typo, *algorith* to *algorithm*.

**Review by srivarra — APPROVED** [review]
Looks Good!

File: README.md [hunk 2]
```diff
@@ -23,12 +23,16 @@ The repo has four main parts, with associated code and jupyter notebooks for eac
...
 
 ### 3. Evaluating a MIBI run
-We have found it useful to generate certain QC plots for each MIBI run to ensure everything went smoothly. The notebooks in this section will automate the generation of these metrics to give a quantitative measure of what happened during the run. They are currently under development, more details coming soon. 
+There are a number of different computational tasks to complete once a MIBI run has finished to ensure everything went smoothly. 
+
+- 3a: real time monitoring. The [MIBI monitoring](./templates/3a_monitor_MIBI_run.ipynb) notebook will monitor an ongoing MIBI run, and begin processing the image data as soon as it is generated. This notebook is being continually be updated as we move more of our processing pipeline to happen in real time as the data is generated.
+- 3b: post-run monitoring. For each step in the monitoring notebook, we have a dedicated notebook that can perform the same tasks once a run is complete. This includes [the image extraction notebook](./templates/extract_bin_file.ipynb) and the [qc metrics notebook](./templates/3b_generate_qc_metrics.ipynb). 
+
```
[the remark this answers is not in the document]
Comment (ngreenwald) [reply README.md:35]: Yup, waiting until the notebook is updated before officially changing the name. Just created an issue
https://github.com/angelolab/toffy/issues/98

**Comment by camisowers** [comment]
I'll make sure to keep the variable names consistent in the MPH notebook too. Looks good!

**Pull request merged**